src/libs/RequestThrottle.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'underscore';
+import CONST from '../CONST';
+
+let requestWaitTime = 0;
+
+function clear() {
+    requestWaitTime = 0;
+}
+
+/**
+ * @returns {Number} time to wait in ms
+ */
+function getRequestWaitTime() {
+    if (requestWaitTime) {
+        requestWaitTime = Math.min(requestWaitTime * 2, CONST.NETWORK.MAX_RETRY_WAIT_TIME);
+    } else {
+        requestWaitTime = _.random(CONST.NETWORK.MIN_RETRY_WAIT_TIME, CONST.NETWORK.MAX_RANDOM_RETRY_WAIT_TIME);
+    }
+    return requestWaitTime;
+}
+
+/**
+ * @param {Number} time
```

Comment:
🔪

Reply:
Oh good catch thanks! That was a left over from testing another approach. Updated.